.gitignore

```diff
@@ -0,0 +1,2 @@
+# add all references that we don't want on github
+.idea/
```

Comment:
Looking good! Just one more addition, just add .DS_Store in the .gitignore, I will ready to merge it